packages/react-core/src/utils/index.ts

```diff
@@ -1,6 +1,6 @@
 import isEmpty from 'lodash/isEmpty';
```

Comment:
We don't do `isEmpty` this time?

Reply:
Was intending of doing `isEmpty`, `isNil` and `isEqual` as a group next.